project/SparkBuild.scala

```diff
@@ -486,9 +485,11 @@ object SparkParallelTestGrouping {
   )
 
   private val DEFAULT_TEST_GROUP = "default_test_group"
+  private val HIVE_EXECUTION_TEST_GROUP = "hive_execution_test_group"
```

Comment:
@gengliangwang, do you know if the group names are displayed for anywhere else? or is it just purely grouping purpose internally?

Reply:
I haven't noticed that it is displayed in somewhere.